stubs/fpdf2/fpdf/actions.pyi

```diff
@@ -1,26 +1,29 @@
+from _typeshed import Incomplete
 from abc import ABC
-from typing import Any
+
+from .syntax import PDFObject
 
 class Action(ABC):
-    def dict_as_string(self) -> None: ...
+    def __init__(self, next_action: PDFObject | str | None = ...): ...
+    def dict_as_string(self, key_values: dict[str, Incomplete] | None = ...) -> None: ...
```

Comment:
This returns whatever a call to `create_dictionary_string` returns: https://github.com/PyFPDF/fpdf2/blob/be3ad07b7ae9c7d39866aa130de177b43fc3f073/fpdf/actions.py#L23

And `create_dictionary_string` always returns `str`: https://github.com/PyFPDF/fpdf2/blob/be3ad07b7ae9c7d39866aa130de177b43fc3f073/fpdf/syntax.py#L99

```suggestion
    def dict_as_string(self, key_values: dict[str, Incomplete] | None = ...) -> str: ...
```